### PR TITLE
Add consensus logic to 'clients' request

### PR DESCRIPTION
**Pull request by ShubhankarS**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -52,8 +53,9 @@ function adapter(uri, opts) {
 
   var prefix = opts.key || 'socket.io';
   var subEvent = opts.subEvent || 'messageBuffer';
-  var requestsTimeout = opts.requestsTimeout || 5000;
+  var requestsTimeout = opts.requestsTimeout || 1000;
   var withChannelMultiplexing = false !== opts.withChannelMultiplexing;
+  var consensusPercentage = isNaN(parseFloat(consensusPercentageFromEnv)) ? parseFloat(consensusPercentageFromEnv) : 1;
```
Comment (parthg-wysa) [inline index.js]:
```suggestion
  var consensusPercentage = !isNaN(parseFloat(consensusPercentageFromEnv)) ? parseFloat(consensusPercentageFromEnv) : 1;
```

Comment (parthg-wysa) [reply index.js]: We also need to be careful with the terminology. It's written percentage, but actually, we are treating it as a number from 0 - 1. A Minor point that might create confusion while setting the value of the env variable

**Review by parthg-wysa — CHANGES_REQUESTED** [review]

**Review by parthg-wysa — COMMENTED** [review]
The overall idea looks good.

File: index.js (hunk 1 above)
Comment (ShubhankarS) [reply index.js]: done

**Review by parthg-wysa — DISMISSED** [review]

**Review by rakeshmakam — DISMISSED** [review]

**Review by paul3412 — DISMISSED** [review]

**Review by parthg-wysa — APPROVED** [review]

**Review by paul3412 — APPROVED** [review]

**Pull request merged**